smee/internal/dhcp/handler/reservation/option.go

```diff
@@ -50,6 +50,13 @@ func (h *Handler) setDHCPOpts(_ context.Context, _ *dhcpv4.DHCPv4, d *data.DHCP)
 	if h.SyslogAddr.Compare(netip.Addr{}) != 0 {
 		mods = append(mods, dhcpv4.WithOption(dhcpv4.OptGeneric(dhcpv4.OptionLogServer, h.SyslogAddr.AsSlice())))
 	}
+	if len(d.ClasslessStaticRoutes) > 0 {
+		if routeBytes, err := encodeClasslessStaticRoutes(d.ClasslessStaticRoutes); err == nil {
+			mods = append(mods, dhcpv4.WithGeneric(dhcpv4.OptionClasslessStaticRoute, routeBytes))
```

Comment:
FYI, there is a [`dhcpv4.OptClasslessStaticRoute`](https://github.com/insomniacslk/dhcp/blob/d3abe7ccb0ad34d401030e9e1365c83075732563/dhcpv4/option_routes.go#L99) function that could be useful here.

Reply:
Thanks for the hint, I have replaced it with that struct completely.